LFS171x/sawtooth-material/sawtooth-tuna/README.md

```diff
@@ -4,3 +4,4 @@ We will be looking at a sample blockchain application that interfaces with Hyper
 
 This is an example of a simple Hyperledger Sawtooth blockchain application written in JavaScript. Hyperledger Sawtooth Tuna Chain allows a user to create named assets (tuna fish) and transfer them between different holders designated by a public key. 
 
+This application is based on (https://github.com/delventhalz/transfer-chain-js). Big thanks to Zac Delventhal, maintainer on Hyperledger Sawtooth.
```

Comment:
Is it based on or is it a copy. If the latter, then we need to say so. My understanding was that it was identical to Zac's code.

Reply:
It seems that the only thing in this directory that is different is the README from Zac's repository.

Reply:
rephrased